spec/tasks/maintenance_stats_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "maintenance stats" do
+  describe "gather_mainttenance_stats" do
+    let(:repository) { create(:repository, full_name: "chalk/chalk") }
+    let!(:auth_token) { create(:auth_token) }
+    let!(:project) do
+      repository.projects.create!(
+        name: "test-project",
+        platform: "Maven",
+        repository_url: "https://github.com/librariesio/libraries.io",
+        homepage: "https://libraries.io"
+      )
+    end
+
+    before do
+      # set the value for DateTime.current so that the queries always have the same variables and can be matched in VCR
+      allow(DateTime).to receive(:current).and_return(DateTime.parse("2018-12-14T17:49:49+00:00"))
+    end
+
+    let!(:tempfile) do
+      Tempfile.new(["temp", ".csv"]).tap do |file|
+        CSV.open(file, "wb") do |csv|
```

Comment:
```suggestion
        CSV.open(file, "wb", col_sep: "\t") do |csv|
```

Match input file format